TVC_MVP/CountyPresenterProtocol.swift

```diff
@@ -8,24 +8,18 @@
 
 import Foundation
 
-protocol CountyPresnterProtocol: class {
+protocol CountyPresenterProtocol: class {
 
     init(view: CountyViewProtocol)
 
     func viewDidLoad()
 
-    func reloadData()
-
     func numberOfSections() -> Int
 
     func numberOfRowsInSection() -> Int
 
-    func removeCounty(at indexPath: IndexPath)
-
-    func fetchCounty(for indexPath: IndexPath) -> County?
-
-    func presentAlert(at indexPath: IndexPath)
+    func getObject(at indexPath: IndexPath) -> County?
 
-    func parseJSON()
+    func didSelectRow(indexPath: IndexPath)
```

Comment:
We should aim to keep the naming consistent. When using the `indexPath` you used the label `at` in other method, here should be the same.